client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/index.tsx

```diff
@@ -16,20 +14,16 @@ const noop = () => {};
 
 const EmailProvidersStackedCard: FunctionComponent< ProviderCard > = ( props ) => {
 	const {
-		additionalPriceInformation,
```

Comment:
Since that prop is no longer required, we should no longer pass it when this component [is](https://github.com/Automattic/wp-calypso/blob/8269366ce8a30c7307282728b529b5e208b7b8dc/client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx#L258) [used](https://github.com/Automattic/wp-calypso/blob/8fe0c3e5dae5d2d3e19fd9b09d9bf252801ba666/client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx#L244).